assets/stylesheets/style.scss

```diff
@@ -23,15 +23,11 @@
 	//@import 'shared/welcome'; // welcome messages
 	//@import 'shared/infinite-scroll-end'; // Last page marker once infinite scroll has reached end
 
-	@import 'shipping-services'; // Shipping Services Groups, Group and Entry Component styles
-	@import 'shipping-label'; // Shipping Labels specific styles
-
 	// Main
 	@import 'main'; // global layout and responsive styles
 
 	// Components
 	@import 'components';
-	@import 'components/payment-logo/style';
 
 	// Breakpoints
 	$breakpoints: 480px, 660px, 780px, 960px, 1040px; // adds wp-admin specific breakpoints
```

Comment:
Good catch! I've removed the 780px breakpoint in https://github.com/Automattic/woocommerce-services/pull/1051/commits/e028d3afc2469ef861e3cbfa4fa2253319207a3e